database/src/main/postgres/flows/V0.3.0.1__get_flow_checkpoints.sql

```diff
@@ -132,7 +134,13 @@ BEGIN
                      JOIN flows.partitioning_to_flow PF ON C.fk_partitioning = PF.fk_partitioning
             WHERE PF.fk_flow = i_flow_id
               AND (i_checkpoint_name IS NULL OR C.checkpoint_name = i_checkpoint_name)
-            ORDER BY C.process_start_time DESC
+            ORDER BY
+                CASE
+                    WHEN i_latest_first THEN C.process_start_time
+                    END DESC,
+                CASE
+                    WHEN NOT i_latest_first THEN C.process_start_time
+                    END ASC
```

Comment:
This is tricky on NULL.
Would use an ELSE statement. And probably flip them. So the default is still DESC

Reply:
Or just use a local variable at the beginning
`_latest_first := coalesce(i_latest_first, TRUE);`